tools/ci_build/build.py

```diff
@@ -612,6 +612,7 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs, enab
             executables = ['onnxruntime_test_all.exe']
             if args.build_shared_lib:
                 executables.append('onnxruntime_shared_lib_test.exe')
+                executables.append('onnxruntime_global_thread_pools_test.exe')
             run_subprocess(['vstest.console.exe', '--parallel', '--TestAdapterPath:..\\googletestadapter.0.17.1\\build\\_common', '/Logger:trx','/Enablecodecoverage','/Platform:x64',"/Settings:%s" % os.path.join(source_dir, 'cmake\\codeconv.runsettings')] + executables,
```

Comment:
This isn't dependent on building with the shared lib enabled is it?

Also, does the CI that does code coverage  also need to know about this new test exe?

Reply:
This is dependent on building with shared_lib since the test uses the C APIs. Not sure about the code coverage setup. I can add it wherever code coverage needs to know about this test.